UI/Web/src/app/volume-detail/volume-detail.component.ts

```diff
@@ -613,6 +613,7 @@ export class VolumeDetailComponent implements OnInit {
         });
         break;
       case Action.AddToReadingList:
+        this.actionService.addVolumeToReadingList(this.volume!, this.seriesId, () => {/* No Operation */ });
```

Comment:
I believe the callback is nullable, so you should be able to just not pass anything.